### DISCUSSION TITLE
22차 모임 (2024-06-20) 발표 주제 제안

### DISCUSSION THREAD

**Discussion by haksungjang** [OpenChain-KWG/meeting#6] [General]
안녕하세요. OpenChain Korea Work Group 22차 모임이 2024년 6월20일 오후2시부터 CJ 인재원에서 예정되어 있습니다. 

이 때 주제 발표를 원하시는 분은 발표 제목과 간단한 내용을 comment로 남겨주시기 바랍니다.
혹시, (내가 발표하지는 못하지만) 누군가가 공유해주기를 원하시는 주제가 있어도 comment 남겨주세요.

특별히, 이번 모임에서는 주제 발표 세션과 더불어 Lightening Talk를 시도해보려고 합니다. 
Lightening Talk는 특정 주제에 대해 3~5분 가량 가볍게 소개하는 시간입니다. Lightening Talk로 다루고자 하는 주제가 있다면 comment 남겨주시기 바랍니다. 

감사합니다.

**Comment by haksungjang** [comment]
지난 4월24일 Legal Subgroup에서 다뤄졌던 Entr’ouvert v. Orange 소송 케이스를 이번 모임에서도 세션 발표로 소개하기로 하였습니다.
https://openchain-project.github.io/OpenChain-KWG/subgroup/legal/5.-meetings/4rd-meeting/

SK주식회사 조정년 변리사님 감사합니다. ^^

**Comment by haksungjang** [comment]
Lightening Talk

- 'XZ Open Source Attack : 오픈소스 커뮤니티의 신뢰를 이용한 최초의 공급망 공격 사례'에 대한 소개

https://docs.google.com/document/d/1TGds3d9TErWdNcgC0Jw_CYrxpixXRhHl2KMiiVtWzQI/edit#heading=h.5fzr1w9pp1pq

카카오 황은경 파트장님께서 제안해 주셨습니다. 

감사합니다.

**Comment by haksungjang** [comment]
'정부 SW 공급망 보안 가이드라인'이 발표되었는데요, 
https://www.msit.go.kr/bbs/view.do?sCode=user&mId=113&mPid=238&bbsSeqNo=94&nttSeqNo=3184474

이에 대한 소개 발표를 누군가 자원해 주시면 좋겠습니다. 

- 배경
- 시사점
- 기업 오픈소스 담당자가 취해야 할 Action

감사합니다.

**Reply by JustinWonjaePark** [reply]
손들어봅니다 ✋

**Comment by haksungjang** [comment]
Lightening Talk

'Copilot 등 생성형 AI가 만들어내는 code를 저작권 이슈를 최소화하여 사용하는 방법'에 대해 가볍게 다뤄주실 분 계신가요? 

감사합니다.

**Comment by CHAEJINYOUNG** [comment]
발표 제목 : ISO/IEC 18974 준비 현황 공유 - 삼성SDS 채진영/김기륜
발표 내용 : 준비 일정, 정책서 내용, 이슈, 오픈소스 거버넌스 체계 소개 등

**Reply by haksungjang** [reply]
안녕하세요, 채진영 님. 
발표 제안 감사합니다! 혹시 예상하시는 발표 시간은 몇분 정도일까요? :)

**Comment by CHAEJINYOUNG** [comment]
안녕하세요? 삼성SDS 채진영 프로 입니다.

발표 시간은 대략 20분 정도면 될 것 같습니다.
  
관련해서 궁금한 점이 있으시면 언제든지 연락 주십시오. 
감사(感謝) 드립니다.
